src/main/resources/application.yml

```diff
@@ -30,11 +30,25 @@ micronaut:
     intercept-url-map:
       - pattern: "${kafkahq.server.base-path:}/static/**"
         access: "isAnonymous()"
+     # Ldap configuration
```

Comment:
Please remove the whole file modification, not concerning the feature

Reply:
Removed changes